index.html

```diff
@@ -1179,29 +1170,27 @@
 tracking, a form of [=cross-context recognition=], but it can also be used for other
 inappropriate information flows.
 
-## Data minimization {#data-minimization}
+## Data Minimization {#data-minimization}
 
 <div class="practice">
-<span class="practicelab">Sites, user agents and other parties should minimize the amount of
+<span class="practicelab">Sites, user agents, and other parties should minimize the amount of
 <a>personal data</a> they transfer between parties on the Web.</span>
 </div>
 
-Data minimization limits the risks of data being disclosed or misused and also makes it possible to
+Data minimization limits the risks of data being disclosed or misused, and also makes it possible to
 more meaningfully provide people with understandable decisions about data about them.
 
 <div class="practice">
 <span class="practicelab">Web APIs should be designed to minimize the amount of data that sites need
-to request to carry out their users' goals and provide granularity and user controls over <a>personal
+to obtain to carry out their users' goals and provide granularity and user controls over <a>personal
```

Comment:
I think "request" was right here. Or maybe "get access to". A lot of sites won't actually "obtain" the extra information that an API call returned, they'll just let it get immediately GC'ed.

Reply:
The thing that trips me up with "request" is that if the sites _need_ (as it says line above) the information to carry out the users' goals they have to actually get it, not just ask for it?